cli/info/info.go

```diff
@@ -32,8 +32,15 @@ func getInfoFromRemote(c cli.Config) (resp *info.Resp, err error) {
 	req.Label = c.Label
 	req.Profile = c.Profile
 
-	serv := client.NewServer(c.Remote)
-
+	cert, err := helpers.LoadClientCertificate(c.MutualTLSCertFile, c.MutualTLSKeyFile)
```

Comment:
with default "" certFile and keyFile, this always errs out? May need some protection.